src/components/orders/OrdersUserDetailsTable/index.tsx

```diff
@@ -53,37 +115,69 @@ interface RowProps {
 
 const RowOrder: React.FC<RowProps> = ({ order, isPriceInverted }) => {
   const { creationDate, buyToken, buyAmount, sellToken, sellAmount, kind, partiallyFilled, shortId, uid } = order
+  const [_isPriceInverted, setIsPriceInverted] = useState(isPriceInverted)
+
+  useEffect(() => {
+    setIsPriceInverted(isPriceInverted)
+  }, [isPriceInverted])
+
+  const invertLimitPrice = (): void => {
+    setIsPriceInverted((previousValue) => !previousValue)
+  }
 
   return (
     <tr key={shortId}>
       <td>
-        {
+        <HeaderTitle>
+          Order ID <HelpTooltip tooltip={tooltip.orderID} />
+        </HeaderTitle>
+        <HeaderValue>
           <RowWithCopyButton
             className="span-copybtn-wrap"
             textToCopy={uid}
             contentsToDisplay={
-              <Link to={`/orders/${order.uid}`} target="_blank">
+              <Link to={`/orders/${order.uid}`} rel="noopener noreferrer" target="_blank">
```

Comment:
Not for this PR

Just realized would be good to use the same network component as with addresses.

Right now it works like this, but we unnecessarily query all networks before loading the order. But we already know the network it belongs to.